.editorconfig

```
@@ -0,0 +1,15 @@
+# EditorConfig is awesome: https://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+[*]
+indent_style = space
+indent_size = 4
+end_of_line = crlf
```

We can change this parameter to be more cross-platform friendly
```suggestion
end_of_line = lf
```